<?xml version="1.0" ?>
<robot name="arbie">

<!-- Arbitrary urdf file so the controllers don't complain -->

<link name="base_link">
</link>

<joint name="arm1" type="continuous">
    <parent link="base_link"/>
    <child link="link1"/>
    <origin rpy="0 0 0" xyz="0.1 0 0"/>
</joint>
<link name="link1">
</link>

<joint name="arm2" type="continuous">
    <parent link="link1"/>
    <child link="link2"/>
    <origin rpy="0 0 0" xyz="0.1 0 0"/>
</joint>
<link name="link2">
</link>

<joint name="arm3" type="continuous">
    <parent link="link2"/>
    <child link="link3"/>
    <origin rpy="0 0 0" xyz="0.1 0 0"/>
</joint>
<link name="link3">
</link>

<joint name="wrist_yaw" type="continuous">
    <parent link="link3"/>
    <child link="link4"/>
    <origin rpy="0 0 0" xyz="0.1 0 0"/>
</joint>
<link name="link4">
</link>

</robot>
